tests/pipeline/nodes/output/test_media_writer.py

```diff
@@ -64,7 +65,13 @@ def test_writer_writes_single_video(self, writer, create_video):
         video = create_video(size, nframes=20)
         for frame in video:
             writer.run({"filename": "test.mp4", "img": frame, "fps": 30})
-        assert directory_contents() == set(["test.mp4"])
+        
+        #pattern to check for time stamp filename_DDMMYY-hh-mm-ss.extension
+        #approved extension = ["jpg", "jpeg", "png", "mp4", "avi", "mov", "mkv"]
+        #listed in input.recorded.py
+        pattern = r".*_\d{6}-\d{2}-\d{2}-\d{2}\.[a-z0-9]{3,4}$"
```

Comment:
The `[a-z0-9]{3,4}` is a little liberal because we have an expected list of approved extensions. Can be good to have that hardcoded list here (for now).

Reply:
a hardcoded check on the extension used in the test case is done prior to checking the regex pattern of the saved filename

Reply:
thanks for the changes